src/main/kotlin/org/elm/lang/core/psi/PsiElement.kt

```diff
@@ -72,12 +73,106 @@ fun <T : PsiElement> PsiElement.parentOfType(vararg classes: KClass<out T>): T?
 inline fun <reified T : PsiElement> PsiElement.contextOfType(strict: Boolean = true): T? =
         PsiTreeUtil.getContextOfType(this, T::class.java, strict)
 
-inline fun <reified T : PsiElement> PsiElement.childOfType(strict: Boolean = true): T? =
-        PsiTreeUtil.findChildOfType(this, T::class.java, strict)
+inline fun <reified T : PsiElement> PsiElement.descendantOfType(): T? =
```

Comment:
Love this rename; I've had to look through the source code multiple times to remember whether this was only direct children or not.

Reply:
I know, I can't believe that they have built so much code on such poor naming conventions. There were undoubtedly a lot of bugs along the way.